ddclient.in

```diff
@@ -811,6 +814,7 @@ sub main {
     $SIG{'HUP'}  = sub { $caught_hup  = 1; };
     $SIG{'TERM'} = sub { $caught_term = 1; };
     $SIG{'INT'}  = sub { $caught_int  = 1; };
+
```

Comment:
Undo this line add. (It might be better to have this blank line, but this PR isn't the right place to add it.)